frontend/packages/dev-console/src/components/import/section/ResourceSection.tsx

```diff
@@ -22,12 +22,12 @@ type ResourceSectionProps = {
 
 const createHelpText = (k8sModel: K8sKind, helpText: string) => {
   return (
-    <>
+    <div style={{ lineHeight: '1em' }}>
```

Comment:
The easiest way to know when to use `lineHeight` is (almost) never :)

What you have done here is take 2 paragraphs and try to condense them down to 1 font-size in line height. The browser is stopping you from doing this to the full extent :) 

What was the goal here? Just to make them more condensed?

Reply:
Yes, because in UXD it seems the spacing between them is less than what is shown here by default. So thought of reducing line height. You are right though, its more of a hack :P

Reply:
Recommend going with a more css approach, turning both `<p>` into `<div>`s and using a Patternfly spacer `var(--pf-global--spacer--sm)` on the top one. Instead of a `line-height` :)

Reply:
done :)